apps/web/components/question/Question.tsx

```diff
@@ -49,7 +57,7 @@ const getQuestionComponent = ({
   if (isWalkthroughItemTypeMultiChoice(questionData.walkthroughItemType)) {
     return (
       <QuestionMultiChoice
-        value={value}
+        value={value as string}
```

Comment:
Do we anticipate this being a persistent pattern? Curious if there's a better way to handle this mutation?

Reply:
I don't think. I'm hoping I don't have to do it once I redo state at a higher level so I have to pass less things around and everything can just subscribe to what it needs. I'm starting that migration with the footer ticket.